metrics/register.go

```diff
@@ -0,0 +1,38 @@
+package metrics
+
+import (
+	"fmt"
+
+	"go.opencensus.io/stats/view"
+)
+
+var registeredViews = map[string][]*view.View{}
+
+func RegisterViews(name string, views []*view.View) error {
```

Comment:
Godocs missing everywhere.

Reply:
Added godocs, let me know if they are not clear enough